types/js-yaml/index.d.ts

```diff
@@ -9,7 +9,12 @@
 
 export as namespace jsyaml;
 
-export function load(str: string, opts?: LoadOptions): object | string | number | null | undefined;
+// tslint:disable-next-line:no-unnecessary-generics
+export function load<T = unknown[] | Record<string, unknown> | string | number>(
```

Comment:
In this case, a generic isn't any safer than an assertion. It's basically an alternate, safer-looking syntax for the same thing.

Reply:
Fair enough. I'll remove it.